--- lib/injection_container.config.dart
+++ lib/injection_container.config.dart
@@ -107,11 +107,7 @@
     gh.lazySingleton<_i102.LocalNotificationService>(
       () => _i102.LocalNotificationService(),
     );
     gh.lazySingleton<_i361.Dio>(() => registerModule.dio);
-    gh.lazySingleton<_i558.FlutterSecureStorage>(
-        () => registerModule.secureStorage);
     gh.lazySingleton<_i454.SupabaseClient>(() => registerModule.supabaseClient);
-    gh.lazySingleton<_i558.FlutterSecureStorage>(
-        () => registerModule.secureStorage);
     gh.lazySingleton<_i895.Connectivity>(() => registerModule.connectivity);
     gh.lazySingleton<_i111.MiniplayerHeightNotifier>(
